### ISSUE TITLE
Improve input/output panel UI

### ISSUE THREAD

**Issue by AlexBxl** [tokens-studio/graph-engine#548]
## Overview

This is really ugly and looks like a code editor to me, even if there's "Outputs" in the title I couldn't figure out at first glance that his was the generated outputs (which are super useful! as a new user I want this very visible)
Give me multiple ways to look at this output

![Image](https://github.com/user-attachments/assets/53af01ec-290b-45cc-8d95-f32d82497279)

**Comment by six7** [comment]
Agree. I think Roman had mocks for this, I think we could start to chip away at his designs. DM'ing Roman to ask for mocks (he's not on GH)

**Comment by six7** [comment]
Closing this as this is being handled in https://hyma-team.slack.com/archives/C08MY2D7D89 - we'll create a dedicated issue for it then
